blockchain/types/transaction.go

```diff
@@ -829,6 +829,26 @@ func TxDifference(a, b Transactions) (keep Transactions) {
 	return keep
 }
 
+// FilterTransactionWithBaseFee returns a list of transactions for each account that filters transactions
+// that are greater than or equal to baseFee.
+func FilterTransactionWithBaseFee(pending map[common.Address]Transactions, baseFee *big.Int) (map[common.Address]Transactions, error) {
+	txMap := make(map[common.Address]Transactions)
+	for addr, list := range pending {
+		txs := list
+		for i, tx := range list {
+			if tx.GasPrice().Cmp(baseFee) < 0 {
+				txs = list[:i]
+				break
+			}
+		}
+
+		if len(txs) > 0 {
```

Comment:
What if the first transaction is not satisfied in the above condition?
In this case len(txs)==0, then txMap is not updated. Is this the intended behavior?

Reply:
Yes, I intended this behavior. 
Since this list is sorted by nonce(only call `commitNewWork()`), if the first transaction does not satisfy the above conditions, the next transactions also must not be included.